rust/private/utils.bzl

```diff
@@ -152,7 +152,7 @@ def _expand_location(ctx, env, data):
             env = env.replace(directive, "${pwd}/" + directive)
     return ctx.expand_location(env, data)
 
-def expand_locations(ctx, env, data):
+def expand_env_locations(ctx, env, data):
```

Comment:
Kind ask: now that you touched it, could you improve the documentation to have Args and Returns, and so there's a symmetry between this function and `expand_arg_locations`? Could you also add that this function is meant to be used to expand `$(location //foo)` macros in dict values, and that the `expand_arg_location` is for list elements?

Now that I'm thinking, what do you think about renaming these functions from `env` to `dict_values` and `arg` to `list_elements`?

Reply:
I think both of these functions have docs for Args and Returns, if I understand correctly.

I did the rename.

Reply:
> Could you update docs of rule attributes that are affected by this PR to mention that macros are expanded?

This part is yet to be done.

Reply:
> Now that I'm thinking, what do you think about renaming these functions from `env` to `dict_values` and `arg` to `list_elements`?

With such verbose names, I wonder if it's better to have kept `expand_locations` and checked for whether a list or dict was passed, but that's just a small gripe 😅

Reply:
Now documented (please unresolve if I misunderstood the Args and Returns situation).